Sources/SwiftParser/TokenSpecSet.swift

```diff
@@ -144,7 +144,6 @@ enum ContextualDeclKeyword: TokenSpecSet {
   case dynamic
   case final
   case indirect
-  case infix
```

Comment:
Why did you need to remove `infix` from `ContextualDeclKeyword`?

Reply:
That was a wrong modification, it has been modified.